### ISSUE TITLE
comentarios avances 3 y readme

### ISSUE THREAD

**Issue by AliciaMstt** [almamelisa/ProyectoFinalBioinfo-2018-II#4]
Bien, pero puedes mejorarlo.

En el listado de avances del repo de la clase te faltó poner https://github.com/almamelisa/ProyectoFinalBioinfo-2018-II/blob/master/Avance_3.md entre paréntesis, por favor corrige esto.

No es necesario que pongas los md de los avances. Pero si quieres hacerlo, pon lo links de los avances utilizando el formato que usamos en el listado de proyectos finales, es decir [Avance_1.md](https://github.com/almamelisa/ProyectoFinalBioinfo-2018-II/blob/master/Avance_1.md) en vez de 

Avance_1.md https://github.com/almamelisa/ProyectoFinalBioinfo-2018-II/blob/master/Avance_1.md

En data tienes que especificar y describir qué datos hay en cada folder. 

**importante** tu script maestro TIENE RUTAS ABSOLUTAS. Como hemos visto en clase, esto no es recomendable, por lo que te pido que las sustituyas por rutas relativas. Contará para la calificación.

Correcciones inglés:

to obtain a phylogenetic -> in the data and scripts needed for obtaining a phylogenetic
The using data -> The data

**Comment by almamelisa** [comment]
Ya hice los cambios.

**Comment by AliciaMstt** [comment]
Hola,

Gracias por leer y hacer los cambios. Sin embargo: 

`cd /your/working/dir/trabajo_final/sratoolkit.2.9.0-mac64/bin` sigue teniendo una ruta absoluta. Para quitarla tienes que usar la variable que ya definiste antes en tu script. Por ejemplo:

```
WORKING_DIR=/your/working/dir

cd $WORKING_DIR/trabajo_final/sratoolkit.2.9.0-mac64/bin
./prefetch --option-file ../../SRR_Acc_List.txt

```
 
Por favor atiende este comentario para tu entrega final.

**Comment by almamelisa** [comment]
Ok, gracias Alicia. 
llevo como tres días tratando de instalar jmodeltest para obtener mis modelos, y aún no lo logro, busqué un docker disponible y tampoco. Me pregunto si habrá oportunidad de cambiar el enfoque del trabajo, en lugar de hacer filogenia, realizar otros análisis como estadísticos de heterocigosis, o algo así. 

Ya estoy desesperada!!!

**Comment by AliciaMstt** [comment]
Sí, sin problema con el cambio de enfoque. Por eso les pido avances, para que se den cuenta de cuánto se tarda uno en instalar cosas.
